www/css/ionic.app.css

```diff
@@ -9277,6 +9277,10 @@ body * {
   .sign-in :-ms-input-placeholder {
     color: #f2f2f2; }
 
+.trade #tradeContractChart {
```

Comment:
Do not update .css files directly, please revert the commit and apply changes to scss files